CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## 0.44.0
+
+### ✨ Features and improvements
+
+* The CSP policy of a page using mapbox-gl-js no longer needs to include `script-src 'unsafe-eval'` [#559](https://github.com/mapbox/mapbox-gl-js/issues/559)
+* Add `LngLatBounds#isEmpty()` method [#5917](https://github.com/mapbox/mapbox-gl-js/pull/5917)
+* Coalesce GeoJSON "loadData" requests. This prevents high-frequency calls to setData from building up a backlog. [#5902](https://github.com/mapbox/mapbox-gl-js/pull/5902)
```

Comment:
Err, #5902 got rolled back and https://github.com/mapbox/mapbox-gl-js/pull/5988 hasn't gone back in yet to re-implement it.